packages/apputils/src/dialog.tsx

```diff
@@ -639,6 +702,14 @@ export namespace Dialog {
      */
     button: IButton;
 
+    /**
+     * State of the dialog checkbox.
+     *
+     * #### Notes
+     * It will be null if no checkbox is defined for the dialog.
+     */
+    isChecked: boolean | null;
```

Comment:
I prefer to be explicit that to extrapolate; hence the null value. Especially because there is a possibility that the dialog will be discarded without being opened when the user have another one already open.

In such case, I would rather have the code telling me the checkbox status is `null` than a value not validated by the user.